frontend/views/components/Graphs/PieChart.vue

```diff
@@ -1,53 +1,43 @@
 <template lang='pug'>
 .c-container
-  svg.c-piechart(viewbox='-1 -1 2 2' :style='{ height: size }')
+  svg.c-piechart(viewBox='-1 -1 2 2' aria-hidden='hidden')
```

Comment:
Interesting viewbox. Could you had a comment please?

Reply:
Yes, sure, there's a comment below with a URL explaining more about how this piechart was made:
// Learn more about SVG & PieCharts
// -> https://hackernoon.com/a-simple-pie-chart-in-svg-dbdd653b6936